transforms: 
  - 
    header: 
      seq: 0
      stamp: 
        secs: 0
        nsecs: 0
      frame_id: ar_marker_0
    child_frame_id: static_frame
    transform: 
      translation: 
        x: 0.0
        y: 0.1673
        z: 0.0
      rotation: 
        x: 0.0
        y: 0.0
        z: -0.70710678
        w: 0.70710678
